src/bin/pg_dump/dump_babel_utils.c

```diff
@@ -223,6 +223,33 @@ dumpBabelGUCs(Archive *fout)
 	destroyPQExpBuffer(qry);
 }
 
+/*
+ * serverSupportsConstraintUsingIndex:
+ * Helper function to determine if remote server supports constraint
+ * using index. It is supported on version 15.8 higher and 16.4 or higher.
+ */
+static bool
+serverSupportsConstraintUsingIndex(Archive *fout)
```

Comment:
We should remove this check for old versions, it would be better to fail rather than going with old behavior where we loose information about constraint ordering.

Reply:
Discussed offline, removed the check in newer revision.